IRremoteESP8266.cpp

```diff
@@ -216,25 +216,28 @@ void IRsend::sendNEC (unsigned long data, int nbits, unsigned int repeat) {
   }
 }
 
-void IRsend::sendLG (unsigned long data, int nbits) {
+void IRsend::sendLG (unsigned long data, int nbits, unsigned int repeat) {
+  // Args:
+  //   data:   The contents of the command you want to send.
+  //   nbits:  The bit size of the command being sent.
+  //   repeat: The number of times you want the command to be repeated.
+
```

Comment:
I come from a Python background, and that's where our style guide had them. If the c++ style guide and/or IDEs use them in a different, I'm more than happy to comply. Any pointers to where I can find  how to lay them out for c++ IDEs?